src/dune_engine/dune_rpc_client.ml

```diff
@@ -0,0 +1,63 @@
+module Dune_rpc = Dune_rpc_private
```

Comment:
The code in this module seems completely generic, not really related to action runners. Should this be moved out of this PR for clarity?

Reply:
Done in https://github.com/ocaml/dune/pull/7120